src/commons/report.py

```diff
@@ -104,25 +109,79 @@ def update_tests_status(input_dict: dict, corio_start_time: datetime, value: dic
     # Reason of the test execution failure.
     test_failed = kwargs.get("test_failed", '')
     test_start_time = corio_start_time + value['start_time']
-    if datetime.now() > test_start_time:
-        input_dict["START_TIME"] = f"Started at {test_start_time.strftime('%Y-%m-%d %H:%M:%S')}"
-        if datetime.now() > (test_start_time + value['min_runtime']):
-            pass_time = (test_start_time + value['min_runtime']).strftime('%Y-%m-%d %H:%M:%S')
-            input_dict["RESULT_UPDATE"] = f"Passed at {pass_time}"
-            total_execution_time = value['min_runtime'] if sequential_run else datetime.now(
-            ) - test_start_time
-        else:
-            # Report In Progress, Fail, Aborted and update status.
-            if input_dict["TEST_ID"] in terminated_tests:
-                LOGGER.error("Test execution terminated due to error in %s.", input_dict["TEST_ID"])
-                input_dict["RESULT_UPDATE"] = "Fail"
-            elif test_failed:
-                input_dict["RESULT_UPDATE"] = "Aborted"
+    start_iterations_count = 0
+    completed_iterations_count = 0
+    fpath = get_test_file_path(corio_start_time, input_dict['TEST_ID'])
+    yaml_file = '/root/corio/workload/master_config.yaml'
```

Comment:
Don't hardcode any path here keep it in constant.py so that it is reusable.

Reply:
did this change.